source/Camera/Basic/CaptureHDRLoop/CaptureHDRLoop.cs

```diff
@@ -0,0 +1,50 @@
+/*
+This example shows how to acquire HDR images from the Zivid camera in a loop
+(while actively changing some HDR settings).
```

Comment:
This example shows how to acquire HDR images from the Zivid camera in a loop, 
with settings from .yml files.